bin/check.sh

```diff
@@ -45,6 +45,30 @@ usage() {
   echo_stderr "  $bin_name --plan-code 24ska01 --datacenters fr,gra,rbx,sbg"
 }
 
+notify_gotify() {
+  local message="$1"
+  if [ -z ${GOTIFY_TOKEN+x} ]; then
```

Comment:
You should also check for existence of `GOTIFY_URL` and `GOTIFY_PRIORITY` variables

Reply:
Added the checks for the other two variables